### ISSUE TITLE
GPU training error with CUDA driver version is insufficient for CUDA runtime version

### ISSUE THREAD

**Issue by sun9700** [apache/incubator-mxnet#4213]
Ubuntu 16.04 CUDA 8 CUDNN5 .1  Driver Version: 375.20 
complie success, but Failed in  mnist example:

INFO:root:Start training with [gpu(0)]
[10:23:55] /home/sun/mxnet/dmlc-core/include/dmlc/./logging.h:235: [10:23:55] src/storage/storage.cc:38: Check failed: e == cudaSuccess || e == cudaErrorCudartUnloading CUDA: CUDA driver version is insufficient for CUDA runtime version
Traceback (most recent call last):
  File "/home/sun/LearnMxnet/mnist1.py", line 115, in <module>
    batch_end_callback=mx.callback.Speedometer(batch_size, 200)  # output progress for each 200 data batches
  File "/usr/local/lib/python2.7/dist-packages/mxnet/model.py", line 811, in fit
    sym_gen=self.sym_gen)
  File "/usr/local/lib/python2.7/dist-packages/mxnet/model.py", line 206, in _train_multi_device
    logger=logger)
  File "/usr/local/lib/python2.7/dist-packages/mxnet/executor_manager.py", line 324, in __init__
    self.slices, train_data)
  File "/usr/local/lib/python2.7/dist-packages/mxnet/executor_manager.py", line 236, in __init__
    input_types=data_types)
  File "/usr/local/lib/python2.7/dist-packages/mxnet/executor_manager.py", line 150, in _bind_exec
    arg_arr = nd.zeros(arg_shape[i], ctx, dtype=arg_types[i])
  File "/usr/local/lib/python2.7/dist-packages/mxnet/ndarray.py", line 876, in zeros
    arr = empty(shape, ctx, dtype)
  File "/usr/local/lib/python2.7/dist-packages/mxnet/ndarray.py", line 612, in empty
    return NDArray(handle=_new_alloc_handle(shape, ctx, False, dtype))
  File "/usr/local/lib/python2.7/dist-packages/mxnet/ndarray.py", line 69, in _new_alloc_handle
    ctypes.byref(hdl)))
  File "/usr/local/lib/python2.7/dist-packages/mxnet/base.py", line 77, in check_call
    raise MXNetError(py_str(_LIB.MXGetLastError()))
mxnet.base.MXNetError: [10:23:55] src/storage/storage.cc:38: Check failed: e == cudaSuccess || e == cudaErrorCudartUnloading CUDA: CUDA driver version is insufficient for CUDA runtime version

Process finished with exit code 1

**Comment by kernel8liang** [comment]
have the same issue. Debugging...

**Comment by kernel8liang** [comment]
I solved this issue. 

in my environment，it's because we first installed ```cuda7.5```, then installed ```cuda8.0``` in same machine. with default, the ```nvcc-7.5``` is using,  after ```cuda``` had installed, ```libcuda.so``` will be copy to ```/usr/local/lib64``` and ```/usr/local/lib```. After cuda8.0 installed,  cuda8.0's  libcuda.so override the cuda7.5's so, and we didn't reboot the machine, so the driver is cuda7.5's driver, and the so is the cuda8.0's so. 

when use corresponding driver to cuda, it's solved.
